README.md

```diff
@@ -10,6 +10,7 @@ Terraform Controller is a Kubernetes Controller for Terraform, which can address
 
 - Alibaba Cloud
 - AWS
+- GCP
```

Comment:
Maybe this is a typo. You should add Azure, right?

Reply:
Correct, thanks:)